### PR TITLE
Added BZ1525538 to 3.7 known issues

### PR DESCRIPTION
**Pull request by ahardin-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1542717

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@bostrt PTAL

File: release_notes/ocp_3_7_release_notes.adoc [hunk 1]
```diff
@@ -2883,6 +2883,11 @@ Deployments Support]
 [[ocp-37-known-issues]]
 == Known Issues
 
+* There needs to be a prompt alerting the user when they are being redirected to a
```
Comment (bostrt) [inline release_notes/ocp_3_7_release_notes.adoc]: change 'a' to 'an'

**Comment by bostrt** [comment]
Hi @ahardin-rh! The solution for BZ1525538 will not be to introduce a prompt to the user but to redirect users to only specific URLs. I'm not sure if the Known Issues mention should change based on this but I will leave that to your discretion. If you need some more details from me, let me know!

**Comment by ahardin-rh** [comment]
@bostrt Oh, thanks for clarifying! This is now updated.

**Comment by bostrt** [comment]
@ahardin-rh I think this change works.

**Comment by ahardin-rh** [comment]
No revision history needed.

**Pull request merged**